spec-main/api-crash-reporter-spec.ts

```diff
@@ -196,19 +196,33 @@ ifdescribe(!isLinuxOnArm && !process.mas && !process.env.DISABLE_CRASH_REPORTER_
   });
 
   ifdescribe(!isLinuxOnArm)('extra parameter limits', () => {
-    it('should truncate extra values longer than 127 characters', async () => {
+    function stitchLongCrashParam (crash: any, paramKey: string) {
+      if (crash[paramKey]) return crash[paramKey];
+      let chunk = 1;
+      if (crash[`${paramKey}__${chunk}`]) {
+        let stitched = '';
+        while (crash[`${paramKey}__${chunk}`]) {
+          stitched += crash[`${paramKey}__${chunk}`];
+          chunk++;
+        }
+        return stitched;
+      }
+      return null;
```

Comment:
```suggestion
      let stitched = '';
      while (crash[`${paramKey}__${chunk}`]) {
        stitched += crash[`${paramKey}__${chunk}`];
        chunk++;
      }
      return stitched;
```

Doesn't seem like other code here uses the distinction between `''` and `null` so I dropped that.